e2e/capacity/transactions.test.ts

```diff
@@ -576,6 +576,52 @@ describe('Capacity Transactions', function () {
         });
       });
 
+      describe('when capacity eligible transaction is from the handles pallet and balance less than ED', function () {
```

Comment:
nit: description implies that the test case is unique to the handles pallet, when in fact it is for any capacity-eligible transaction (just using `claimHandle` as the test case)

```suggestion
      describe('when capacity eligible transaction and balance less than ED', function () {
```